skimage/transform/tests/test_geometric.py

```diff
@@ -354,6 +354,24 @@ def test_fundamental_matrix_inverse():
                         [[0, 1, 0], [0, 1, -1], [0, 1, -1]])
 
 
+def test_fundamental_matrix_inverse2():
+    src = np.array([1.839035, 1.924743, 0.543582,  0.375221,
+                    0.473240, 0.142522, 0.964910,  0.598376,
+                    0.102388, 0.140092, 15.994343, 9.622164,
+                    0.285901, 0.430055, 0.091150,  0.254594]).reshape(-1, 2)
+
+    dst = np.array([1.002114, 1.129644, 1.521742, 1.846002,
+                    1.084332, 0.275134, 0.293328, 0.588992,
+                    0.839509, 0.087290, 1.779735, 1.116857,
+                    0.878616, 0.602447, 0.642616, 1.028681]).reshape(-1, 2)
+
+    tform = estimate_transform('fundamental', src, dst)
+
+    # calculate x' F x for each coordinate; should be close to zero
+    p = np.abs(np.sum(np.column_stack((dst, np.ones(len(dst)))) * tform(src), axis=1))
+    assert np.all(p < 0.01)
```

Comment:
@stefanv could you maybe expand on this a bit more. I'm not really following. Your comment seems similar to Eq. 8.5 \$x'^T F x = 0\$, though there is a transpose missing? Also if it is indeed testing Eq. 8.5 how is this related to the property that \$F^T\$ describes the matrix for the reverse transformation?

Reply:
Yes, the test is named incorrectly! I started out wanting to test both directions, and ended up testing only one.

The convention in H&Z is x containing row-vectors, I think, which is the opposite of how we store them.